help/de/Help_ch_GettingHelp.xml

```diff
@@ -0,0 +1,179 @@
+            </menuchoice>wird der andere Teil dieser Dokumentation, das Tutorial, geöffnet.</para>
+        </listitem>
+
+        <listitem>
+          <para>In den meisten Dialogen führt die <guibutton><accel>H</accel>ilfe</guibutton>
+            Taste direkt zu der Seite, auf der der jeweilige Dialog beschrieben wird.</para>
+        </listitem>
+      </itemizedlist>
+    </para>
+    <para>Das komplette Hilfemenü wird in <xref linkend="AccTree-help-menu"></xref>beschrieben.</para>
+
+    <variablelist spacing="compact">
+      <title>Betriebssystem und Hilfeanzeigeprogramm</title>
+      <para>Abhängig von Ihrem Betriebssystem werden unterschiedliche Programme verwendet, um dieses Hilfedokument
+        anzuzeigen:</para>
+      
+      <varlistentry><term><systemitem class="osname">Linux</systemitem></term>
+        <listitem><simpara>Gnome <application>Yelp</application></simpara></listitem></varlistentry>
+
+      <varlistentry><term><systemitem class="osname">MacOS</systemitem></term>
+        <listitem><simpara>Standard Webbrowser</simpara></listitem></varlistentry>
+
+      <varlistentry><term><systemitem class="osname">Windows</systemitem></term>
+        <listitem><simpara>Windows-Hilfeanzeige</simpara></listitem></varlistentry>
+    </variablelist>
+
+    <para>Lesen Sie die jeweiligen Handbücher, um Hilfe zur Hilfeanzeige zu erhalten.</para> 
+
+    <sect2 id="tutorial">
+      <title>&app; Kurs und Konzepte</title>
+      <!--  -->
+      <para>Das Handbuch ist das Gegenstück zu dieser Hilfe. Es erklärt die in &app; verwendeten Konzepte und beinhaltet 
+        ein Lernprogramm das Sie durch die Verwendung von &app; zur Verwaltung Ihrer Konten führt.
+        Es wird angezeigt, wenn Sie <guibutton>Öffnen des neuen Benutzer-Tutorials</guibutton> in dem
+        <guilabel>Willkommen bei &appname;!</guilabel> Menü wählen.</para>
+
+      <para>Um diese Anleitung manuell zu öffnen, gehen Sie zu <menuchoice><guimenu>Hilfe</guimenu>
+        <guimenuitem>&app;-Kurs und Konzepte</guimenuitem></menuchoice> oder betätigen Sie  
+        <keycombo action='simul'><keycap>Strg</keycap><keycap>H</keycap></keycombo>.</para>
+
+      <important>
+        <para>Es wird dringend empfohlen, dieses Handbuch zu lesen, wenn Sie mit der Nutzung von 
+          &app; neu beginnen oder mit den Konzepten der Buchhaltung nicht vertraut sind.</para>
+      </important>
+    </sect2>
+
+    <sect2 id="wiki">
+      <title>&app; Wiki</title>
+      <!-- Translators: if wiki pages in your language exist, add or replace the links in this section
+          see also https://wiki.gnucash.org/wiki/Special:Categories for your language or region code -->
+      <para>Eine große Anzahl an weniger formeller, dafür aber <emphasis>aktuellerer</emphasis> Dokumentation, sowohl über
+        &app; selbst als auch über dessen Wartung und Entwicklung, finden Sie in dem 
+        <ulink url="&url-wiki;">&app; Wiki</ulink>; dort sollte die Seite
```

Comment:
`<ulink url="url-wiki-de;">` wollen wir. Wirf eine Blick auf die Entitäten in [gnc-docbookx.dtd](https://github.com/Gnucash/gnucash-docs/blob/maint/docbook/gnc-docbookx.dtd)
Also in der nächsten Zeile: url-wiki-faq-de

Reply:
Verstanden und korrigiert.